Client/WebFilters/ContentBlocker/Lists/test-compile-all-lists.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+pushd `dirname $0`
+
+echo "Testing main blockers"
+
+for file in *.json
+do
+  ./does-it-compile.swift "$file"
+done
+
+echo ""
+echo "Testing regional blockers"
```

Comment:
We can remove the tests for these blockers because we no longer have them bundled :)

Should mean that Travis will start passing on this PR

Reply:
Will do, thanks :)